Index.html

```diff
@@ -38,17 +38,209 @@ <h2>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                  
+                    <div class="See-btn">     
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+
+                </div>
+            </div>
+
+            
+        </section>
+
+        <section class="aboutMe">
+            <h1>About me</h1>
+           
+                <p>I can help you build product , feature or website
+                  Look through some of my work and experience! If you like 
+                  what you see and have a project you need coded, don't hesitate
+                   to contact me.
+                </p>
+
+                <div class="See-btn">     
+                    <button type="button" class="see-Resume">Get My Resume</button>
+                </div>
+            
         </section>
 
-       
+        <section class="languages">
+            <div class="languageList">
+
+                <div class="languageDetail">
+                    <div class="imgDiv">
+                    <img src="./Images/elipse.png" alt="Ellipse">
+                    </div>
+
+                    <h1>Languages</h1>
+
+                    <div class="list-language ">
+                        <ul>
+                            <li>JavaScript</li>
+                            <li>Ruby</li>
+                            <li>Html</li>
+                            <li>CSS</li>
+                            
+                        </ul>
+                    </div>
+
+                </div>
+                <div class="languageDetail">
+                    <div class="imgDiv">
+                    <img src="./Images/elispe2.png" alt="Ellipse">
+                    </div>
+
+                    <h1>Frameworks</h1>
+
+                    <div class="list-language">
+                        <ul>
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSpec</li>
+                            <li>Capybara</li>
+                            <li>Selenium</li>
+                        </ul>
+                    </div>
+
+                </div>
+                <div class="languageDetail">
+                    <div class="imgDiv">
+                    <img src="./Images/elipse4.png" alt="Ellipse">
+                    </div>
+
+                    <h1>Skills</h1>
+
+                    <div class="list-language">
+                        <ul>
+                            <li>Codekit</li>
+                            <li>Github</li>
+                            <li>Codepen</li>
+                            <li>Gitlab</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
+
+                </div>
+            </div>
```

Comment:
- [ ] Again, this looks like a list of technologies categories ("Languages", "Frameworks" and "Skills"). I suggest you make it a list (`ul`) and each of them in a `<li>` tag. e.g. 
```html
<ul>
  <li>Languages
    <ul> 
      <li>HTML</li>
      <li>CSS</li>
      <li>JavaScript</li>
    </ul>
  </li>
  <li>Frameworks</li>
  <li>Skills</li>
<ul>
```
- [ ] I strongly suggest you align these technology categories with flexbox. Although it works well as it is, you'll appreciate using flexbox when you start working on the desktop view version. 
- [ ] And of course, do not forget to remove blank lines that can be seen as unnecessary, based on my comments in the summary :arrow_up: